bundles/org.openhab.binding.sonos/README.md

```diff
@@ -166,3 +167,14 @@ sitemap demo label="Main Menu"
 		}
 }
 ```
+
+sonos.rules:
```

Comment:
For consistency, either rename this to demo.rules, or rename the others to sonos.things, sonos.items and sonos.sitemap, respectively.

Reply:
yes, thanks for the feedback. this is my first addition to the documentation. I'm learning every day :)